MARC.js

```diff
@@ -384,16 +404,9 @@ record.prototype.translate = function(item) {
 						.replace(	//chop off any translations, since they may have repeated $e fields
 							new RegExp('\\' + subfieldDelimiter + 'd.+'), '');
 		title = this.extractSubfields(title, '200');
-		item.title = title.a;
-		if(title.e) {
-			//If the title proper did not end in a punctuation mark, we should add a colon
-			if(item.title.search(/[A-Za-z0-9]\s*/) != -1) {
-				item.title += ':';
-			}
+		item.title = glueTogether(title.a, title.e, ': ');
+		item.title = clean(item.title);
```

Comment:
Just FYI, if (for whatever reason) `title.a` and `title.e` are empty strings `clean` will throw an error, since `item.title` will be `null` (instead of empty string). I haven't looked whether this condition would fail regardless, but if title can be obtained from other fields later on, obviously we don't want it failing here. Even if this would fail, it would probably be better to fail with "no title set..." error (our default), than "replace is not defined for null"  (or whatever it would be).

So, either `clean` should be able to take a `null`, or `glueTogether` should never return a `null` (or you need more sanity checks in between)